src/ui/screens/Parachains/AuctionsScreen.tsx

```diff
@@ -1,239 +1,114 @@
+            <StatInfoBlock title="First - Last">{`${leasePeriod?.first} - ${leasePeriod?.last}`}</StatInfoBlock>
           </View>
-        )}
-      </ScrollView>
-    </Layout>
-  );
-}
-
-type HeaderProps = {
-  count: string;
-  active: boolean;
-  firstLast: {first: string; last: string};
-  stats: {raised: string; raisedPercent: number};
-  duration: {endingAt: BN; remaining: BN; remainingPercent: number};
-};
-
-function Header(props: HeaderProps) {
-  const {firstLast, stats, duration} = props;
-  const {first, last} = firstLast;
-  const {endingAt, remaining, remainingPercent} = duration;
-  const {raised, raisedPercent} = stats;
-
-  const theme = useTheme();
-
-  return (
-    <View>
-      <View style={headerStyle.container}>
-        <View style={globalStyles.alignCenter}>
-          <Caption>Auction</Caption>
-          <Text>{props.count}</Text>
-        </View>
-        <View style={globalStyles.alignCenter}>
-          <Caption>Active</Caption>
-          <Text style={props.active ? {color: theme.colors.success} : undefined}>{props.active ? 'yes' : 'no'}</Text>
-        </View>
-        <View style={globalStyles.alignCenter}>
-          <Caption>First - Last</Caption>
-          <Text>
-            {first} - {last}
-          </Text>
-        </View>
-      </View>
-      <View style={headerStyle.container}>
-        <View style={headerStyle.container}>
-          <View style={globalStyles.alignCenter}>
-            <Caption>Total</Caption>
-            <View style={globalStyles.rowAlignCenter}>
-              <Chart percent={raisedPercent} />
-              <Padder scale={0.5} />
-              <Text>{raised}</Text>
+          <Padder scale={2} />
+          <View style={styles.itemRow}>
+            <View style={globalStyles.flex}>
+              {endingPeriod && (
+                <ProgressChartWidget
+                  title={`Ending period`}
+                  detail={`${endingPeriod.remainingPercent}%\n${endingPeriod.remaining.slice(0, 2).join('\n')}`}
+                  data={[endingPeriod.remainingPercent / 100]}
+                />
+              )}
+            </View>
+            <View style={globalStyles.flex}>
+              <ProgressChartWidget
+                title={`Total Raised`}
+                detail={`${raisedPercent}%\n${raised.split(' ').join('\n')}`}
+                data={[raisedPercent / 100]}
+              />
             </View>
           </View>
-        </View>
-        <View style={headerStyle.container}>
-          <View style={globalStyles.alignCenter}>
-            <Caption>Ending Period</Caption>
-            <View style={globalStyles.rowAlignCenter}>
-              <Chart percent={remainingPercent} />
+        </Card.Content>
+      </Card>
+      <Padder scale={0.5} />
+      <Card style={styles.container}>
+        <Subheading style={globalStyles.textCenter}>{`Winning Bid`}</Subheading>
+        <List.Item
+          title={`${winningBid?.projectName} #${winningBid?.projectId}`}
```

Comment:
Would it be meaningful to show the project Id? If so, we should explain what it is. Just a number won't provide enough details -- thinking about users. I did not understand what the number was at the first go.